src/App.elm

```diff
@@ -0,0 +1,95 @@
+port module App exposing (changeState, createState, newStateMachine, stateButton)
+
+{-| Elm state button based on a simple state machine
+
+
+# StateButton
+
+@docs createState
```

Comment:
@m-sev könntest du bitte zumindest noch für diese öffentlichen Funktionen die Docs in dem Elm-Docs-Format hinzufügen?